recipes/officetopdf/LICENSE.md

```diff
@@ -0,0 +1,70 @@
+Apache
```

Comment:
This file appears to be included in the sources that are downloaded. I think you can delete it and `conda-build` will automatically find it from there instead. This ensures it will never be outdated if the license was to change.